### ISSUE TITLE
Please increase max Kilobits Per Second in Quality tab

### ISSUE THREAD

**Issue by lidarruser** [lidarr/Lidarr#1146]
I want to upgrade my files until they're 24bit/192kHz. The minimum for that would be 4000kbps. But the max is only 1500 when I enable "advanced settings".

So can you please make the maximum 10,000kbps?

In addition, I noticed that the highest quality is WAV. Does renaming the title of a quality change how it searches for that quality? Going to assume if I rename the title to FLAC 192kHz, it'll still only search for .WAV right? So could you add more quality tiers? 24bit/48kHz, 24/96, 24/192, etc?

Rutracker has a 24/192 category (and the others) so I don't think it's too out of the way to suggest it...

Thank you!

**Comment by Qstick** [comment]
Just drag the max to the top for unlimited

**Comment by lidarruser** [comment]
Yes, but I can't choose a minimum of 4000 because 1500 is the max... That's why I'm asking for 10,000 just to be safe.

**Comment by Qstick** [comment]
The issue is it kills scaling for every other quality

**Comment by lidarruser** [comment]
Are talking about scrollbar scaling and how increasing the range by that much will make it harder to choose a specific number? Thing is, it's already really annoying to choose a number this way. So why not just make the advanced setting be enabled by default? Typing>scrollbar.

It'll help all users...

**Comment by Qstick** [comment]
You are one user speaking for 100k, most of which don’t want 24bit albums thus don’t have this issue. It needs to be thought out and done in a way that doesn’t have negative impacts on most of the user base. Average users don’t need decimal level accuracy, close close enough is good enough.....

In the mean time you can type the values in the db manually to get it working how you want.

**Comment by lidarruser** [comment]
I'm not going to respond to your ad-populum argument as that's a huge can of worms I don't want to go into. Vinyls (192kHz) being popular again, blah blah blah, it doesn't matter. If you know that my suggestion will ruin it for everyone else, then I'll go with your decision. If there's a way to get my quality tiers fixed without messing it for everyone else then that's great! Thank you for the suggestion!

Can you go into more detail with what settings you're talking about? When I try typing 4000 into the minimum box, it changes to 1490. Maybe you're talking about something else?

Thank you for responding!

**Comment by romanr** [comment]
> Are talking about scrollbar scaling and how increasing the range by that much will make it harder (or impossible) to choose a specific number? Thing is, it's already really annoying to choose a number this way. So why not just make the advanced setting be enabled by default? Typing>scrollbar. If anything, the scrollbar should be the advanced setting as even with 1500 as the max, you'd need to change your mouse DPI to even get close to the number you want. "Ok I want 320. No not 317.5... Not 321.2... Not... Guess that's close enough.........."
> 
> It reminds me of that meme where you have to manually press "+" starting from 1 until you reach your phone number. "1... 2... 3... 1-555-690-5900, finally."
> 
> It'll help all users...

As long time Lidarr user I want to thank you and contributors for this great software.  

**There's a way to fix this problem with minimal effort and have everybody happy.** 
Please hear me out. If anything, I'm a UX designer with 10+ years of SaaS projects and difficult design decisions like this one.

![image](https://user-images.githubusercontent.com/329079/224513154-ef1cb10a-07a0-4bf2-968d-7cdbc6a0bdb8.png)

The argument here is about ruining the experience of bitrate adjustment range slider for all users.
Let's ask first, the function of adjusting bitrate for quality settings - what is the use case? 
For vast majority of users, they will only go to that screen, as OP (or me) did, because we reached the limit of allowed bitrate. Not because we wanted to play with adjustments and create very specific quality rules.
This range slider was added in times when audio standards were still not settled like in 2013 when this project started. 
And possibly main reason for this range slider is because it allows to visualise the bitrate distribution without effort in integrating charts or drawing those lines on canvas. 
When Lidarr started there was many different rates and formats. The download speeds were slower and costly. I remember upgrading music to 320kbps and how huge those files were at that time.
Look what we have now. It's either 320kbps or Lossless (and now 24bit Lossles). 

Now this limitation creates more problems. It is misleading to display "Unlimited" when it's in fact limited by max value of 1500. 
![image](https://user-images.githubusercontent.com/329079/224513201-4901f8ce-d91d-4fce-b01a-6dd4bda22e06.png)
I'm sure that not only me, but many others wasted time trying this and that, wondering why it's not "unlimited" until realising that it was from the times when 1500 was so large that nobody would imagine reaching that limit.

## options:

1. As it's famously said, great design is saying "No" nine times out of ten. Let's say No to the slider. Remove and change Min/Max inputs from Advanced to Basic settings mode.

2. Another option is to add `disabled` attribute to range sliders. They'll still visualise the bitrates just not responding to touch. And enabling Min/Max for all settings modes.
The quality management screen itself is one of most "advanced" screens and having those inputs there is understandable.

3. Future proof option: add "unlimited" checkbox at max rate. No need for sliders or that can be disabled. This is most time consuming option

![image](https://user-images.githubusercontent.com/329079/224513265-48725f7c-7d21-4aa1-a05e-65f883226dca.png)


4. Last option: not really the option. as more people move to High Res Lossless this issue will loom more and more. Flood the issue tracker and take contributors time to read reports, spending time again on same discussion.

**Comment by azukaar** [comment]
It's much simpler than that, you could just have a different max for Flac that goes up to say 10k, and keep the others at 1.5k. There's no reason for the max to be coupled anyway

**Comment by alucard91** [comment]
Here is another scenario which might add more reasons to address topic issue:
When library contains 24-Bit albums they are falling in "Cut-Off Unmet" category since they are on higher side of qualuty cutoff which is already odd. But also basically it's killing entire cutoff management since i can't bulk upgrade my library without downgrading this 24-Bit albums. Maybe increasing quality ceiling for lossless formats to 4000 will not be a great deal at the end?

**Comment by alucard91** [comment]
Sad to hear that, theese projects are trully amazing when tailored to needs.
Huge respect to ones, making it possible

**Comment by Surely1244** [comment]
@bakerboy448 I think there's been a misunderstanding of the issue here. This is a bug. Setting the max quality to "unlimited" actually sets the max quality to 1.5Mbps, at least in the "Cutoff Unmet" tab. I have multiple 24-bit FLAC files marked as cutoff unmet with bitrates higher than 1.5Mbps despite having unlimited set under quality.

**Comment by bakerboy448** [comment]
Bitrate settings within a single quality tier don't affect download ranking or quality cutoffs—check out the FAQ for more details on how quality profiles work. Also, I'm no longer involved with Servarr apps and am not aware of any active Developers working on the project.

**Comment by Surely1244** [comment]
Thank you, appreciate you taking the time to respond.
